packages/lesswrong/components/localGroups/LocalGroupPage.tsx

```diff
@@ -286,35 +363,39 @@ const LocalGroupPage = ({ classes, documentId: groupId }: {
       />
       {topSection}
       <SingleColumnSection>
-        <SectionTitle title={`${group.inactive ? "[Inactive] " : " "}${group.name}`}>
```

Comment:
The changes to this group name area are so that the action buttons on the right (ex. subscribe) flow under the location on mobile, instead of squeezing the group name.